source/ways-of-working/environments/aks-auto-shutdown.html.md.erb

```diff
@@ -0,0 +1,57 @@
+---
+title: AKS Auto Shutdown
+last_reviewed_on: 2023-02-07
+review_in: 6 months
+weight: 10
+---
+
+# <%= current_page.data.title %>
+
+### Schedule
+
+See below the list of AKS clusters that have auto-shutdown enabled alongside their schedule:
+
+| Shared Services Environment | CFT Environment | Schedule |
+|-|-|-|
+| Demo | Demo | 6:30pm |
```

Comment:
Isn't it 8pm it shuts down? https://github.com/hmcts/aks-auto-shutdown/blob/fd130c813239b53ed991641c97c796eb246a09be/.github/workflows/aks-auto-shutdown.yaml#L5

and then started by 7am?

Reply:
Yeah I was a bit silly with this part apologies let me take another look